### PR TITLE
Add some initial framekwork stuff

### PR DESCRIPTION
**Pull request by beklapko**
## Description

- Start filling out the ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
-# stash-playground
-👷 Samples for Stash SDK
+<a id="readme-top"></a>
```
Comment (beklapko) [inline README.md:1]: View a preview here: https://github.com/stashgg/stash-playground/blob/b7da78589436ddd5079fd62644f9378e2afabfe9/README.md

File: README.md [hunk 2]
```diff
@@ -1,2 +1,70 @@
...
+
+[![Contributors][contributors-shield]][contributors-url]
```
Comment (beklapko) [inline README.md:3]: I added some badges here but which ones do you want to see?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,70 @@
...
+    <a href="https://docs.stash.gg/docs/demo-walkthrough">View a demo</a>
+    ·
+    <a href="https://github.com/stashgg/stash-playground/issues/new">Report an issue or make a request</a>
+    ·
+    <a href="https://dev.stash.gg/">Stash website</a>
+    ·
+    <a href="https://www.linkedin.com/company/stashgg/">LinkedIn</a>
+</div>
+
+## About the Stash Playground
```
Comment (beklapko) [inline README.md:33]: Are there any sections here that I'm missing?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,70 @@
...
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![YouTube][youtube-shield]][youtube-url]
+
+<br />
+<div align="center">
+  <a href="https://github.com/othneildrew/Best-README-Template">
+    <img src="images/logo_horizontal_black.png" alt="Logo" width="80" height="80">
```
Comment (beklapko) [inline README.md:13]: The logo is all messed up right now. Need to probably find a different asset and size it correctly. Definitely open to suggestion for this!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,70 @@
...
+
+Stash provides Unity and Unreal SDKs for working with our APIs. This means you can build Stash functionality into the same project you build your game in.
+
+## Backend examples
+
+Some Stash integrations require you to build backend APIs. These are often for approving and processing purchases. The backend examples cover different frameworks for building these APIs and integrating with Stash. It's possible you have related logic in place already.
+
+## Frontend examples
+
+Need help with this section
```
Comment (beklapko) [inline README.md:51]: Is there anything specific you want me to call out in this description?

File: README.md [hunk 6]
```diff
@@ -1,2 +1,70 @@
...
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![YouTube][youtube-shield]][youtube-url]
+
+<br />
+<div align="center">
+  <a href="https://github.com/othneildrew/Best-README-Template">
```
Comment (beklapko) [inline README.md:12]: The logo + top of the ReadMe needs some more tweaking but this is a little better than what I had initially.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,70 @@
...
+    <a href="https://github.com/stashgg/stash-playground/issues/new">Report an issue or make a request</a>
+    ·
+    <a href="https://dev.stash.gg/">Stash website</a>
+    ·
+    <a href="https://www.linkedin.com/company/stashgg/">LinkedIn</a>
+</div>
+
+## About the Stash Playground
+
+This repo contains various framework examples you can use with Stash. There are frontend, backend, and game SDK examples. Stash regularly adds to and updates this repo. If you have any issues with a code example, or want to see other examples, please let us know!
```
Comment (beklapko) [inline README.md:35]: This isn't the final text in any of these sections, just threw something in there quickly so we can start looking at the structure of the ReadMe.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,78 @@
...
+</div>
+
+> [!IMPORTANT]
+> This repo is currently being built out.
+
+## About the Stash Playground
+
+This repo contains various framework examples you can use with Stash. There are frontend, backend, and game SDK examples. Stash regularly adds to and updates this repo. If you have any issues with a code example, or want to see other examples, please let us know.
+
+We have a prioritized set of frameworks that we've created examples for. It's difficult maintaining examples for every framework, so we've narrow the scope so that we can provide quality examples for popular frameworks. If you think there's something critical missing from these examples, let us know and we'll look into adding it.
```
Comment (beklapko) [inline README.md:43]: We should actually define what these priorities are.

**Review by oliexe — APPROVED** [review]

**Pull request merged**